domains/SignIn/index.js

```diff
@@ -150,6 +151,11 @@ const SignIn = ({ navigation }) => {
               </View>
               <Text style={styles.passwordText}>{I18n.t('signIn.showPassword')}</Text>
             </View>
+            <Autofill
```

Comment:
I assume this is here just for testing right? I'd merge master into this branch and make sure Autofill works in the patientIDForm. Thanks

Reply:
Yes that was just for testing @hopetambala